server/api-manager.js

```diff
@@ -39,7 +40,10 @@ export default class MindbodyQueries {
     );
     this.requestNum++;
     if (!this.atLimit()) {
-      return request.makeRequest();
+      let requestPromise = request.makeRequest();
+      //todo: Store the new authToken so that class users don't need to?
```

Comment:
This would be handier than the current system where the user must store the authToken for the class. We'd also need to update the "sample usage" code at the bottom of the file to match this.